src/fetchers/stats-fetcher.js

```diff
@@ -99,6 +126,34 @@ const totalCommitsFetcher = async (username) => {
   return 0;
 };
 
+const totalStarsFetcher = async (username, repoToHide) => {
```

Comment:
Can you maybe add a docstring to this new function? TIP: [GitHub's co-pilot](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code) makes this very easy:

https://user-images.githubusercontent.com/17570430/193562723-003a2f7a-f0dd-4c51-a47a-2da23111a7b1.mp4

Reply:
✅ added